docs/xmlrpc_client.py

```diff
@@ -0,0 +1,77 @@
+import xmlrpc.client
```

Comment:
This change doesn't belong here

Reply:
Before the `docs` folder was inside the module, isn't it correct that I removed it from the module?